packages/linting/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@atmina/linting",
+  "version": "3.0.0",
```

Comment:
Is this a breaking change still actually? Since it's not necessary to update your own eslint.config.js in the project anymore and it appears to be backwards compatible?

Reply:
Leaving aside that any change in linting rules is technically a breaking change:

We already removed ESLint 8 from the test matrix and I updated the peer dependencies to reflect this. So even if no action is required by the end user, it might be a good idea to have a clear visual indicator that this is the "ESLint 9" update.

Reply:
Alright, fair! True, dropping support is usually indicated as breaking change, in the back of my mind, we still did 👌